src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml

```diff
@@ -33,12 +33,7 @@
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-full">
                     <div data-test-id="go-back-section" class="nhsuk-back-link">
```

Comment:
Do we still need the `data-test-id="go-back-section"` attribute?

Reply:
👍